src/diffusers/pipelines/audioldm2/modeling_audioldm2.py

```diff
@@ -0,0 +1,1491 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
```

Comment:
I think it's okay to not add documentation for this one as we don't expect it to be reused by other pipelines unline `UNet2DConditionModel`.